### ISSUE TITLE
IFR

### ISSUE THREAD

**Issue by giovannic** [mrc-ide/hypatia#26]
Update to at least squire v0.5.3 to resolve this

**Comment by cwhittaker1000** [comment]
tl;dr OJ and I have actually managed to get all the relevant parameters into squire recently, and so as long as these are ported over from squire, I think this issue's basically sorted! Updated parameters are available here - they are the parameters in `default_probs` here: https://github.com/mrc-ide/squire/blob/master/R/run.R#L4

This issue is about ensuring that the most up-to-date version of the model parameters are being used in hypatia, and through this, ensuring we recapitulate the Infection Fatality Ratio (probability of death conditional on getting COVID-19) matches up to that estimated recently in Brazeau et al (https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis/covid-19/report-34-ifr/). 

Currently, only hospitalised individuals die and so the key parameters dictating the IFR are `prob_hosp` (probability of requiring hospitalisation given disease, age-specific), `prob_severe` (probability of requiring ICU given disease and hospitalisation, age-specific), `prob_non_severe_death_treatment` (probability of dying given hospitalisation but not severe enough to warrant an ICU stay, age-specific) and `prob_severe_death_treatment` (probability of dying given hospitalisation for severe cases who require an ICU stay, age-specific).

OJ and I got updated versions of these parameters (as well as others) into squire recently and so all that's required here is to port over these parameters. They are the parameters in `default_probs` here: https://github.com/mrc-ide/squire/blob/master/R/run.R#L4

Note that `prob_non_severe_death_no_treatment` and `prob_severe_death_no_treatment` are the corresponding probabilities of dying given a particular disease severity, but for individuals who do not receive appropriate healthcare, due to capacity being overwhelmed.

**Comment by ceeb950** [comment]
Giovanni has already added these probabilities to hospitalisation.R in hypatia and get_parameters gets the SQUIRE parameters anyway, so all requirements are covered. Closes #26
